### ISSUE TITLE
Date format issue when citing a book

### ISSUE THREAD

**Issue by dondreab** [dsifford/academic-bloggers-toolkit#404]
**ABT Version:** 4.12.1

**PHP Version:** 5.6

Divi

**Plugin List:**
- Plugin 1
- Plugin 2

Which web browser are you using (including version)
Firefox 57.0.3

Inputting date of 1997 would work

error given about date format - screenshot attached

<img width="960" alt="screenshot 2017-12-31 21 10 56" src="https://user-images.githubusercontent.com/34994273/34465585-b03b1800-ee6f-11e7-98a9-b8eff4be8047.png">

**Comment by dsifford** [comment]
Thanks for the report.

This is a duplicate of #401. It's fixed already but there's still a couple things I need to finish before pushing out the update.

**Comment by dondreab** [comment]
Thank you!!!! Will watch for the update - really need this plugin (and appreciate it)

**Comment by dsifford** [comment]
@dondreab 

The update is still not ready for wordpress.org because there are lots of pretty drastic changes to how custom citation styles are handled that I need to document and test more thouroughly test, but you can download a copy of a beta version from here: https://github.com/dsifford/academic-bloggers-toolkit/releases/tag/v4.13.0-beta

This should resolve your issue with dates.

**Comment by dondreab** [comment]
This is the error I get when I try to add a book after searching by ISBN:
An unexpected error occurred

TypeError: t.people is undefined

Please report this error, including the steps taken to trigger it, here: 
https://github.com/dsifford/academic-bloggers-toolkit/issues

**Comment by dsifford** [comment]
@dondreab Give this one a try: https://github.com/dsifford/academic-bloggers-toolkit/releases/tag/v4.13.0-beta.1

**Comment by dondreab** [comment]
Well - now, although it requires a publisher name it:

1) does not grab the publisher name
2) when you paste in the publisher name, it stays blank and since it requires it, it won't let you post the reference.

Sorry :( (and thank you for all the work!)

Dee

**Comment by dsifford** [comment]
Can you explain the steps you're taking in greater detail so that I might be able to reproduce the issue on my end?

Thanks for your patience.

**Comment by dondreab** [comment]
When i add a book - i do an isbn search - it fills in the blanks - except for the publisher for some reason.

So, I look up publisher, put the publisher name in the slot, but it doesn't 'take' - that slot stays blank.

And since a name in the publisher slot is required in order to save the reference, i can't save that book reference.

Hope that makes sense!

**Comment by dsifford** [comment]
I see. Thanks for the clarification.

Reason why some books don't show publisher information: I'm using a very, very mediocre API that only offers information for a fraction of books. Most other APIs are pay-to-use or require login information that I can't distribute out as a plugin.

Reason why the field doesn't "take" -- likely a bug. I'll look into it when I get time.

**Comment by dsifford** [comment]
should be fixed in `v4.13.0`
